bigtable-hbase-1.x-parent/bigtable-hbase-1.x-mapreduce/README.md

```diff
@@ -79,17 +120,46 @@ from/to Cloud Bigtable and import HBase snapshots using a Map Reduce cluster (ie
    
     # Import from HBase snapshot
     gcloud dataproc jobs submit hadoop \
+        --project <project-id> \
         --cluster <dataproc-cluster> \
         --region <dataproc-region> \
-        --jar bigtable-hbase-1.x-mapreduce-2.0.0-alpha1-hadoop.jar \
+        --jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
         -- \
         import-snapshot \
         -Dgoogle.bigtable.project.id=<project-id> \
         -Dgoogle.bigtable.instance.id=<instance-id> \
         <snapshot-name> \
         <snapshot-dir> \
         <table-id> \
-        <tmp-dir>   
+        <tmp-dir>
+   
+    # HashTable on HBase
+    gcloud dataproc jobs submit hadoop \
+        --project <project-id> \
+        --cluster <dataproc-cluster> \
+        --region <dataproc-region> \
+        --jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        -- \
+        hash-table \
+        -Dhbase.zookeeper.quorum=<source-zk-quorum> \
+        <table-id> \
+        <hash-outputdir-hbase>
+   
+    # SyncTable on Bigtable
+    gcloud dataproc jobs submit hadoop \
+        --project <project-id> \
+        --cluster <dataproc-cluster> \
+        --region <dataproc-region> \
+        --jar bigtable-hbase-1.x-mapreduce-2.5.0-shaded-byo-hadoop.jar \
+        -- \
+        sync-table \
+        --dryrun=true \
+        --sourcezkcluster=<source-zk-quorum> \
+        --targetbigtableproject=<project-id> \
+        --targetbigtableinstance=<instance-id> \
+        <hash-outputdir-hbase> \
+        <source-table-id> \
+        <target-table-id>
```

Comment:
Please validate all the commands, I am not very familiar with Hadoop commands and have not run them.

Reply:
updated test class to validate key-value arguments. other args may be set as strings and we perform basic validation on number of args